pkg/thanos/statefulset_test.go

```diff
@@ -354,25 +354,16 @@ func TestLabelsAndAlertDropLabels(t *testing.T) {
 				"test": "test",
 			},
 			AlertDropLabels:         nil,
-			ExpectedLabels:          []string{`test="test"`},
-			ExpectedAlertDropLabels: []string{},
+			ExpectedLabels:          []string{`test="test"`, `thanos_ruler_replica="$(POD_NAME)"`},
+			ExpectedAlertDropLabels: []string{"thanos_ruler_replica"},
 		},
 		{
 			Labels: map[string]string{
 				"test": "test",
 			},
 			AlertDropLabels:         []string{"test"},
-			ExpectedLabels:          []string{`test="test"`},
-			ExpectedAlertDropLabels: []string{"test"},
-		},
-		{
-			Labels: map[string]string{
```

Comment:
I would still keep a test case when the user has configured additional and dropped labels properly.

Reply:
Ok, done.